qiskit_sphinx_theme/translations.py

```diff
@@ -26,34 +26,34 @@ def setup(app):
 def _extend_html_context(app, config):
```

Comment:
I kept changes as minimal as possible to this file. All I changed was where we get the values out of config so that it's easier to unit test the functions. No logic changes.